monitoring/rid_qualifier/create_flight_record_from_kml.py

```diff
@@ -152,24 +162,20 @@ def generate_flight_record(
             alt=coordinates[2],
             accuracy_h=flight_description.get('accuracy_h'),
             accuracy_v=flight_description.get('accuracy_v'),
-            # TODO: need to check value for extrapolated
+            # TODO: extrapolated ?
             extrapolated=False,
             )
-        # TODO: RIDHeight values ?
         aircraft_height = RIDHeight(distance=ALTITUDE_AGL, reference="TakeoffLocation")
```

Comment:
Let's actually omit the `RIDHeight` since it is optional and it will be tricky to compute reasonably-correct values for it.  That will also make it so we don't need `ALTITUDE_AGL`.

Reply:
Getting error on removing RIDHeight value
raise ValueError('Expected to find dictionary data to populate {} object but instead found {} type'.format(parse_type.__name__, type(source).__name__))
ValueError: Expected to find dictionary data to populate RIDHeight object but instead found NoneType type